recipes/auto-dotenv/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: auto-dotenv
+  version: 1.0
+
+build:
+  skip: true  # [win]
```

Comment:
My guess is that this one could be noarch generic and installable on Windows for those who use the Windows bash, right?

Reply:
That seems right to me! I updated to `noarch: generic`.